### DISCUSSION TITLE
Manual run of ECS Deploy Runner to upgrade the service catalog

### DISCUSSION THREAD

**Discussion by pete0emerson** [gruntwork-io/knowledge-base#340] [Help] [answered]
We are trying to update our service catalog to the latest version. I have a question about this change https://github.com/gruntwork-io/terraform-aws-service-catalog/releases/tag/v0.79.0. It says:

> The security group for the ecs-deploy-runner now defaults to using name_prefix instead of name, which means that the security group will have a randomized suffix for its name. When updating to this module, the security group will be recreated using the new name, along with the ECS Task Definition for the runner, which will cause a disruption in actively running ECS Deploy Runner tasks. This means that the ECS Deploy Runner will not be able to self deploy this change.

Does this mean that we should do something manually? If so, do you have documentation or an example on how to do that?

**Comment by adokarasev** [comment, marked as answer]
Basically it means that you should not use ECS Deploy Runner to roll out this update. My mistake was that we use GitHub action and it calls ECS Deploy Runner to execute terraform commands. You have to execute terraform/terragrunt apply from your laptop or from CI if it does not use ECS Deploy Runner
